CHANGELOG.md

```diff
@@ -22,7 +22,6 @@ These changes are available in the [master branch](https://github.com/PrefectHQ/
 ### Fixes
 
 - Fix defaults for unspecified ARNs in the Fargate Agent - [#1634](https://github.com/PrefectHQ/prefect/pull/1634)
-- Fix ShellTask return value on empty stdout - [#1632](https://github.com/PrefectHQ/prefect/pull/1632)
```

Comment:
why did this get removed?

Reply:
I have no idea, my changelog is really messed up for some reason